docs/ecosystem/panel.md

```diff
@@ -1,108 +1,221 @@
 # Panel
 
-[Panel](https://panel.holoviz.org/) is a tool to build interactive web applications and dashboards using Python code.
+[Panel](https://panel.holoviz.org/) is a framework for building interactive tools for notebooks, dashboards, and web applications that can run on a server—all using Python.
 
-Panel [has been reported to work](https://github.com/developmentseed/lonboard/issues/262) with Lonboard. However, it appears that Panel [does not support reactive updates](https://github.com/holoviz/panel/issues/5921) in the same way that [Shiny](./shiny.md) does, so the map will necessarily be recreated from scratch on every update.
+## Prerequisites
 
-## Example
+To run the code below, you need to install the following packages:
 
-This example was written by [@MarcSkovMadsen](https://github.com/MarcSkovMadsen) in [issue #262](https://github.com/developmentseed/lonboard/issues/262).
+```bash
+pip install panel colorcet ipywidgets_bokeh geopandas palettable lonboard pyogrio
+```
 
-```py
-"""Panel data app based on https://developmentseed.org/lonboard/latest/examples/north-america-roads/"""
-# pip install panel colorcet ipywidgets_bokeh geopandas palettable lonboard
-import colorcet as cc
+## Tutorial
+
+In this tutorial, you will learn how to display a `lonboard` `Map` via the [`IPyWidget`](https://panel.holoviz.org/reference/panes/IPyWidget.html) *pane*.
+
+Create a file named `app.py` with the following content:
+
+```python
 import geopandas as gpd
+from lonboard import Map, ScatterplotLayer
+import panel as pn
 
-from lonboard import Map, PathLayer
-from lonboard.colormap import apply_continuous_cmap
-from palettable.palette import Palette
+pn.extension("ipywidgets")
+
+@pn.cache
+def get_data():
+    return gpd.read_file(gpd.datasets.get_path("naturalearth_cities"))
+
+gdf = get_data()
+layer = ScatterplotLayer.from_geopandas(gdf, radius_min_pixels=2, get_fill_color="red")
+cities_map = Map(layer, _height=500)
```

Comment:
Is this `_height` parameter required here? Does it work without it? Ideally the Map would automatically take up the full height of the `div` provided by Panel, but that might be tricky.

Reply:
I have removed the use of `_height` and replaced it with `map.layout.width=map.layout.height="100%"`. It will work better for both fixed and responsive sizing modes. I've done this in all 3 examples.